plugins/katello/nightly/user_guide/content_view_import_export/index.md

```diff
@@ -6,19 +6,19 @@ version: nightly
 
 # Content View Import/Export
 
-Content view import/export is one of the new features in Katello 3.9. This feature is for users who want the *exact* same content view version on one Katello instance to be available on another Katello instance. The other Katello instances may or may not have Internet access.
+* Content view import/export is one of the new features in Katello 3.9. This feature is for users who want the *exact* same content view version on one Katello instance to be available on another Katello instance. The other Katello instances may or may not have Internet access.
```

Comment:
Was wondering why the shift from paragraphs to bullet points?

Reply:
@ehelms I was trying to make it look a bit more formatted and nicer. I am fine with keeping it like the rest of the docs, I was just trying something new wrt formating.

Reply:
I get your thinking. For me, it starts to feel like bullet point spam as these are not lists of something or if they are they lack some intro context. For example, if oyu added a sentence or two that led into what import/export was and that it has changed from previous versions in the following ways followed by this list that would gel better.